.storybook/gnosisTheme.js

```diff
@@ -1,7 +1,7 @@
 import { create } from '@storybook/theming/create';
 
 export default create({
-  base: '#f7f5f5',
+  base: '#F6F7F8',
```

Comment:
It probably makes sense to export this from a central colours file or something similar.

Reply:
Why is that not used in the other places this PR touches then?

Reply:
I see now that the most of them are svgs. However, `src/theme.ts` can use this. Or vice versa.

Reply:
I see now that the most of them are svgs. However, src/theme.ts can use this. Or vice versa.